### PR TITLE
Initial server based on screwdriver-api

### PR DESCRIPTION
**Pull request by jer**
Implements npm start

### CODE REVIEW

**Comment by nkatzman** [comment]
👍  looks good to me

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+## License
+
+Code licensed under the BSD 3-Clause license. See LICENSE file for terms.
+
+[npm-image]: https://img.shields.io/npm/v/screwdriver-api-aws.svg
+[npm-url]: https://npmjs.org/package/screwdriver-api-aws
+[downloads-image]: https://img.shields.io/npm/dt/screwdriver-api-aws.svg
+[license-image]: https://img.shields.io/npm/l/screwdriver-api-aws.svg
+[issues-image]: https://img.shields.io/github/issues/screwdriver-cd/screwdriver-api-aws.svg
+[issues-url]: https://github.com/screwdriver-cd/screwdriver-api-aws/issues
```
Comment (tkyi) [inline README.md]: https://github.com/screwdriver-cd/api-aws/issues

File: package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "screwdriver",
+    "yahoo"
+  ],
+  "license": "BSD-3-Clause",
+  "author": "Jeremiah Wuenschel <jeremiah.wuenschel@gmail.com>",
+  "contributors": [
+    "Dao Lam <daolam112@gmail.com>",
+    "Darren Matsumoto <aeneascorrupt@gmail.com>",
+    "Jeremiah Wuenschel <jeremiah.wuenschel@gmail.com>",
+    "Noah Katzman <nkatzman@yahoo-inc.com>",
```
Comment (tkyi) [inline package.json]: @nkatzman update your email?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][wercker-image]][wercker-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
+
+> Screwdriver API running in AWS
+
+## Usage
+
+```bash
+npm install screwdriver-api-aws
+```
+
```
Comment (stjohnjohnson) [inline README.md]: Want to add:
```
npm start
curl http://localhost:8666/v3/status
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,31 @@
...
+[npm-image]: https://img.shields.io/npm/v/screwdriver-api-aws.svg
+[npm-url]: https://npmjs.org/package/screwdriver-api-aws
+[downloads-image]: https://img.shields.io/npm/dt/screwdriver-api-aws.svg
+[license-image]: https://img.shields.io/npm/l/screwdriver-api-aws.svg
+[issues-image]: https://img.shields.io/github/issues/screwdriver-cd/screwdriver-api-aws.svg
+[issues-url]: https://github.com/screwdriver-cd/screwdriver-api-aws/issues
+[wercker-image]: https://app.wercker.com/status/cbebea1cd675a32ed57ab64748244ea2
+[wercker-url]: https://app.wercker.com/project/bykey/cbebea1cd675a32ed57ab64748244ea2
+[daviddm-image]: https://david-dm.org/screwdriver-cd/screwdriver-api-aws.svg?theme=shields.io
```
Comment (stjohnjohnson) [inline README.md]: Ditto, `api-aws` remove the screwdriver

**Comment by FenrirUnbound** [comment]
👍

**Comment by tkyi** [comment]
👍 LGTM

**Pull request merged**